packages/file_selector/file_selector_web/test/utils_test.dart

```diff
@@ -14,15 +11,9 @@ void main() {
     group('acceptedTypesToString', () {
       test('works', () {
         final List<XTypeGroup> acceptedTypes = <XTypeGroup>[
```

Comment:
Throughout the changes in this package, can't these lists be const instead of each item in the list?

Reply:
Yes, you are right. It's doable and better.